### DISCUSSION TITLE
《崩坏：星穹铁道》简评

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#21] [Announcements]
# 《崩坏：星穹铁道》简评

一支有战斗力的队伍，必须是有「浓度」的。

https://zexwoo.blog/posts/gossip/honkai-star-rail-review/

**Comment by JourneyHua** [comment]
四月获取了 15781 星琼 145 张车票
五月获得了 14781 星琼 47 张车票
六月截止26日获得了 10387 星琼 41 张车票捏

**Comment by xiele88787** [comment]
问下，开头的角色的高清立绘是哪里弄的呢？网上找不到这么高清的

**Reply by ZexWoo** [reply]
从官网嫖的。
